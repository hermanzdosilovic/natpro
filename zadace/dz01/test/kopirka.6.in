5 3
1000 2 25 500 500
