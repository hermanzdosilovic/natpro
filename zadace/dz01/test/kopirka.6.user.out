1000 / 2 25 / 500 500
